bylaws.md

```diff
@@ -957,52 +957,29 @@ her official capacity and as to action in another capacity while
 holding such office.
 
 **Section 13.6. Insurance.** The corporation shall purchase and 
-maintain insurance on behalf of any person who is or was a director, 
-officer, member, employee or agent of the corporation, or is or was 
-serving at the request of the corporation as a director, officer, 
-employee or agent of another corporation, partnership, joint 
-venture, trust or other enterprise against any liability asserted 
-against him or her and incurred by him or her in any such capacity, 
-or arising out of his or her status as such, whether or not the 
-corporation would have the power to indemnify him or her against 
-such liability under the provisions of this Article or of 
-applicable law. The corporation shall purchase and maintain general 
-liability insurance, supplemented as necessary to cover any special 
-events the corporation may hold.
-
-**Section 13.7. Definitions.** For purposes of this Article, 
-references to "the corporation" shall include, in addition to the 
-resulting corporation, any constituent corporation (including any 
-constituent of a constituent) absorbed in a consolidation or 
-merger which, if its separate existence had continued, would have 
-had power and authority to indemnify its directors, officers, and 
-employees or agents, so that any person who is or was a director, 
-officer, employee or agent of such constituent corporation, or is 
-or was serving at the request of such constituent corporation as 
-a director, officer, employee or agent of another corporation, 
-partnership, joint venture, trust or other enterprise,
-shall stand in the same position under this Article with respect 
-to the resulting or surviving corporation as he or she would have 
-with respect to such constituent corporation if its separate 
-existence had continued, and references to "other enterprises" 
-shall include employee benefit plans; references to "fines" shall
-include any excise taxes assessed on a person with respect to any
-employee benefit plan; and references to "serving at the request
-of the corporation" shall include any service as a director,
-officer, employee or agent of the corporation which imposes duties
-on, or involves services by, such director, officer, employee, or
-agent with respect to an employee benefit plan, its participants, 
-or beneficiaries; and a person who acted in good faith and in a 
-manner he or she reasonably believed to be in the interest of the
-participants and beneficiaries of an employee benefit plan shall be
-deemed to have acted in a manner "not opposed to the best
-interests of the corporation" as referred to in this Article.
+maintain insurance on behalf of any person who is or was
+a director or officer of the corporation or, while a director 
+or officer of the corporation, is or was serving at the request 
+of the corporation as a director, officer, employee or agent 
```

Comment:
Consider making line 963 and line 892 have the same use of commas to avoid ambiguous reading of `employee or agent of another corporation` or `employee, or agent of another corporation`. This may also address @gvanrossum's question too.